components/README.md

```diff
@@ -13,16 +13,16 @@ Every Kyma component resides in a dedicated folder which contains its sources an
 The component's name consists of a term describing the component, followed by the **component type**. The first part of the name may differ depending on the component's purpose.
 This table lists the available types:
 
-| type|description|example|
-|--|--|--|
-|controller|A [Kubernetes Controller](https://kubernetes.io/docs/concepts/workloads/controllers/) which reacts to a standard Kubernetes resource or manages [custom resource definition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) resources. The component's name reflects the name of the primary resource it controls.|Function-controller|
-|controller-manager|A daemon that embeds all [Kubernetes Controllers](https://kubernetes.io/docs/concepts/workloads/controllers/) of a domain. Such an approach brings operational benefits in comparison to shipping all controllers separately. A `controller-manager` takes the name of the domain it belongs to. | - |
-|operator|is a [Kubernetes Operator](https://coreos.com/operators/) which covers the application-specific logic behind the operation of the application, such as steps to upscale a stateful application. It reacts on changes made to custom resources derived from a given [Custom Resource Definition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/). It uses the name of the application it operates. |application-operator|
-|job| A [Kubernetes Job](https://kubernetes.io/docs/tasks/job/) which performs a task once or periodically. It uses the name of the task it performs. |istio-patch-job (not renamed yet)|
-|proxy| Acts as a proxy for an existing component, usually introducing a security model for this component. It uses the component's name. 
-|service| Serves an HTTP/S-based API, usually securely exposed to the public. It uses the domain name and the API it serves.|connector-service|
-|broker| Implements the [Open Service Broker](https://www.openservicebrokerapi.org/) specification to enrich the Kyma Service Catalog with the services of a provider. It uses the name of the provider it integrates with.|azure-broker|
-|configurer| A one-time task which usually runs as an [Init Container](https://kubernetes.io/docs/concepts/workloads/pods/init-containers/) in order to configure the application.|dex-static-user-configurer
+| Type | Description | Example|
+|---|---|---|
+| **controller** | A [Kubernetes Controller](https://kubernetes.io/docs/concepts/workloads/controllers/) which reacts to a standard Kubernetes resource or manages [custom resource definition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) resources. The component's name reflects the name of the primary resource it controls.| Function-controller |
```

Comment:
```suggestion
| **controller** | A [Kubernetes Controller](https://kubernetes.io/docs/concepts/workloads/controllers/) which reacts to a standard Kubernetes resource or manages [Custom Resource Definition](https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/) resources. The component's name reflects the name of the primary resource it controls.| Function-controller |
```

Reply:
That's not what we have in our [guidelines](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#terminology).

Reply:
We have some inconsistency in our guidelines then before one section below the one that you've linked, there's Custom Resource Definition :/ You can see it [here](https://github.com/kyma-project/community/blob/main/docs/guidelines/content-guidelines/02-style-and-terminology.md#capitalized-terms).

Reply:
I know about it (I also thought it a bit weird and wondered whether we should talk about it), but CR and CRD are different concepts (related, but different), so it's not applicable here.